### PR TITLE
Release 20221128

### PR DESCRIPTION
**Pull request by flavioheleno**

### CODE REVIEW

File: app/dependencies.php [hunk 1]
```diff
@@ -177,7 +175,7 @@
       VersionParser::class => autowire(VersionParser::class),
       Twig::class => function (ContainerInterface $container): Twig {
         $cache = false;
-        if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+        if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```
Comment (flavioheleno) [inline app/dependencies.php:178]: @okiedork should this be kept as "production"?

File: app/settings.php [hunk 2]
```diff
@@ -15,21 +15,22 @@
...
-              'dsn' => "pgsql://${_ENV['POSTGRES_USER']}:${_ENV['POSTGRES_PASSWORD']}@${_ENV['POSTGRES_HOST']}:${_ENV['POSTGRES_PORT']}/${_ENV['POSTGRES_DB']}"
+              'dsn' => "pgsql://{$_ENV['POSTGRES_USER']}:{$_ENV['POSTGRES_PASSWORD']}@{$_ENV['POSTGRES_HOST']}:{$_ENV['POSTGRES_PORT']}/{$_ENV['POSTGRES_DB']}"
             ],
             'queue' => [
-              'dsn' => "amqp://${_ENV['AMQP_USER']}:${_ENV['AMQP_PASS']}@${_ENV['AMQP_HOST']}:${_ENV['AMQP_PORT']}"
+              'dsn'      => "amqp://{$_ENV['AMQP_USER']}:{$_ENV['AMQP_PASS']}@{$_ENV['AMQP_HOST']}:{$_ENV['AMQP_PORT']}",
+              'prefetch' => 100
             ],
-            'displayErrorDetails' => (isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'development'),
+            'displayErrorDetails' => (isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'dev'),
```
Comment (flavioheleno) [inline app/settings.php:27]: @okiedork should this be kept as "development"?

File: app/settings.php [hunk 3]
```diff
@@ -15,21 +15,22 @@
...
             'logError'            => true,
             'logErrorDetails'     => true,
             'logger' => [
               'name' => 'slim-app',
               'path' => isset($_ENV['DOCKER']) ? 'php://stdout' : __DIR__ . '/../logs/app.log',
-              'level' => isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'development' ? LogLevel::DEBUG : LogLevel::INFO,
+              'level' => isset($_ENV['PHP_ENV']) === false || $_ENV['PHP_ENV'] === 'dev' ? LogLevel::DEBUG : LogLevel::INFO
```
Comment (flavioheleno) [inline app/settings.php:33]: @okiedork should this be kept as "development"?

File: docker/README.md [hunk 4]
```diff
@@ -90,7 +90,7 @@ POSTGRES_HOST=pph-postgres
 AMQP_USER=<rabbitmq username>
 AMQP_PASS=<rabbitmq password>
 AMQP_HOST=pph-rabbit
-PHP_ENV=development
+PHP_ENV=dev
```
Comment (flavioheleno) [inline docker/README.md:93]: @okiedork should this be kept as "development"?

File: docker/php.Dockerfile [hunk 5]
```diff
@@ -85,17 +82,17 @@ COPY composer.json composer.json
...
 # COMMAND LINE INTERFACE
 #============================================
-FROM php:8.1.6-cli-alpine3.16 as cli
+FROM php:8.1.12-cli-alpine3.16 as cli
 
 # https://blog.packagecloud.io/eng/2017/02/21/set-environment-variable-save-thousands-of-system-calls/
-ENV TZ=:/etc/localtime
-ENV PHP_ENV=development
+ENV TZ=:UTC
+ENV PHP_ENV=dev
```
Comment (flavioheleno) [inline docker/php.Dockerfile:95]: @okiedork should this be kept as "development"?

File: docker/php.Dockerfile [hunk 6]
```diff
@@ -159,23 +160,26 @@ ARG VERSION=latest
...
 # FPM SAPI
 #============================================
-FROM php:8.1.6-fpm-alpine3.16 as fpm
+FROM php:8.1.12-fpm-alpine3.16 as fpm
 
 # https://blog.packagecloud.io/eng/2017/02/21/set-environment-variable-save-thousands-of-system-calls/
-ENV TZ=:/etc/localtime
-ENV PHP_ENV=development
+ENV TZ=:UTC
+ENV PHP_ENV=dev
```
Comment (flavioheleno) [inline docker/php.Dockerfile:182]: @okiedork should this be kept as "development"?

File: phinx.php [hunk 7]
```diff
@@ -12,14 +13,14 @@
   ],
   'environments' => [
     'default_migration_table' => 'phinxlog',
-    'default_environment' => 'development',
-    'production' => [
-      'dsn' => "pgsql://${_ENV['POSTGRES_USER']}:${_ENV['POSTGRES_PASSWORD']}@${_ENV['POSTGRES_HOST']}/${_ENV['POSTGRES_DB']}"
+    'default_environment' => 'dev',
```
Comment (flavioheleno) [inline phinx.php:16]: @okiedork should this be kept as "development"?

File: phinx.php [hunk 8]
```diff
@@ -12,14 +13,14 @@
...
+    'prod' => [
```
Comment (flavioheleno) [inline phinx.php:17]: @okiedork should this be kept as "production"?

File: phinx.php [hunk 9]
```diff
@@ -12,14 +13,14 @@
...
+      'dsn' => "pgsql://{$_ENV['POSTGRES_USER']}:{$_ENV['POSTGRES_PASSWORD']}@{$_ENV['POSTGRES_HOST']}/{$_ENV['POSTGRES_DB']}"
     ],
-    'development' => [
-      'dsn' => "pgsql://${_ENV['POSTGRES_USER']}:${_ENV['POSTGRES_PASSWORD']}@${_ENV['POSTGRES_HOST']}/${_ENV['POSTGRES_DB']}"
+    'dev' => [
```
Comment (flavioheleno) [inline phinx.php:20]: @okiedork should this be kept as "development"?

File: public/index.php [hunk 10]
```diff
@@ -19,7 +19,7 @@
 // Instantiate PHP-DI ContainerBuilder
 $containerBuilder = new ContainerBuilder();
 
-if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```
Comment (flavioheleno) [inline public/index.php:22]: @okiedork should this be kept as "production"?

File: public/index.php [hunk 11]
```diff
@@ -62,7 +62,7 @@
 $routes = require __DIR__ . '/../app/routes.php';
 $routes($app);
 
-if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```
Comment (flavioheleno) [inline public/index.php:65]: @okiedork should this be kept as "production"?

File: app/dependencies.php (hunk 1 above)
Comment (okiedork) [reply app/dependencies.php:178]: Let's use `prod` here.  We are already using it like that.

File: app/settings.php (hunk 2 above)
Comment (okiedork) [reply app/settings.php:27]: Here we use `dev`.

File: app/settings.php (hunk 3 above)
Comment (okiedork) [reply app/settings.php:33]: `dev` here as well.

File: bin/console.php [hunk 12]
```diff
@@ -28,7 +30,7 @@
 // Instantiate PHP-DI ContainerBuilder
 $containerBuilder = new ContainerBuilder();
 
-if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```
[the remark this answers is not in the document]
Comment (okiedork) [reply bin/console.php:33]: `prod` here.

File: docker/README.md (hunk 4 above)
Comment (okiedork) [reply docker/README.md:93]: `dev`.

File: docker/php.Dockerfile (hunk 5 above)
Comment (okiedork) [reply docker/php.Dockerfile:95]: `dev`.

File: docker/php.Dockerfile (hunk 6 above)
Comment (okiedork) [reply docker/php.Dockerfile:182]: `dev`.

File: phinx.php (hunk 7 above)
Comment (okiedork) [reply phinx.php:16]: `dev`.

File: phinx.php (hunk 8 above)
Comment (okiedork) [reply phinx.php:17]: `prod`.

File: phinx.php (hunk 9 above)
Comment (okiedork) [reply phinx.php:20]: `dev`.

File: public/index.php (hunk 10 above)
Comment (okiedork) [reply public/index.php:22]: `dev`.

Comment (okiedork) [reply public/index.php:22]: Ok just kidding, `prod` here.

File: public/index.php (hunk 11 above)
Comment (okiedork) [reply public/index.php:65]: `prod`.

**Pull request merged**